session/tlv.go

```diff
@@ -105,6 +111,10 @@ func SerializeSession(w io.Writer, session *Session) error {
 		tlvRecords, tlv.MakePrimitiveRecord(typeCreatedAt, &createdAt),
 	)
 
+	tlvRecords = append(
+		tlvRecords, tlv.MakePrimitiveRecord(typeRevokedAt, &revokedAt),
```

Comment:
nit: could combine this with the `append` right above it.

Reply:
indeed! will update

Reply:
done :)